packages/react-form/src/hooks/form.ts

```diff
@@ -21,6 +23,9 @@ import {useSubmit} from './submit';
  *     fields: {
  *       title: useField('some default title'),
  *     },
+ *     lists: {
+ *       dynamicList: useDynamicList<Card>([{cardNumber: '4242 4242 4242 4242', cvv: '422'}], emptyCardFactory)
```

Comment:
Currently, the way we get the `dynamicList` is because the `ListBag` takes a dynamicList property. Hence, if you are passing in a dynamicList in the form it needs to be 

```typescript
lists: {dynamicList: useDynamicList}
```

Reply:
I think the card explanation is used throughout the README for dynamic list so wanted to sort of keep it consistent